README.md

```diff
@@ -18,6 +19,97 @@ This is yet another attempt at making a usable programming language.
+        depending on whether you intend to run `python3 a.py` inside `foo/` or
+        `python3 -m foo.a` outside `foo/`. In Oomph, you `import "b.oomph" as b`
+        and it always works.
+    - If you run your code with `python3 -m foo` or `python3 -m foo.thing`,
+        you should to create an empty `foo/__init__.py`. Otherwise, you create an
+        [implicit namespace package](https://www.python.org/dev/peps/pep-0420/). There's
+        no warning, but it's bad practice, and Python will just do something weird when
+        it imports your code.
+    - If `baz.py` does `import foo.bar`, then any other file that imports `foo` will also
+        get `foo.bar`. Then those other files might rely on `foo.bar` without actually
+        importing it, and they break in surprising ways when someone removes
+        `import foo.bar` from `baz.py`.
+- Many beginners think that `10^2 == 100`. In Python, `10^2 = 8` and `10**2 = 100`.
+- I have seen a beginner writing `x**2 + y*+2`, where `*+` should be `**`.
+    It's valid Python code, and Python interprets `+2` as `2`.
+- In Python, `--x` and `++x` are not syntax errors. Both of those evaluate to `x`,
+    without incrementing or decrementing anything.
+- Python's error messages are terrible. For example:
+    - `TypeError: NoneType object is not iterable` means that you are trying to `for` loop
+        over something that is `None` when it shouldn't be. The error message mentions
+        `None` and looping, but not in a way that someone new to Python would understand.
+    - `SyntaxError: invalid syntax` pointing at a valid line of code usually means that
+        the previous line is missing a `)`. This error message points at the wrong line
+        and says nothing about parentheses.
+- There's a `list` builtin in Python, but you can still make a variable named `list`.
+    It works fine at first, but causes confusing errors when you try to convert something
+    to a list with `list(thing)`.
+- Python's `for item in thing: ...` creates an `item` variable set to
+    the last element of `thing`. You won't get an error if you use it after the loop.
+- If you `import string`, then you can't make a variable named `string`. Actually you can,
+    and it will work, until you try to actually use something from the imported module.
+    (Oomph clearly separates imported modules and variables, so a variable named
+    `path` works fine even if you `import "<stdlib>/path.oomph" as path`)
+
+Python gotchas that exist in Oomph (but fixing is planned):
+- Modifying a list while looping over it doesn't behave like you would expect.
+    Try `for item in the_list: the_list.remove(item)` in Python, for example.
+
+
+## Learning Oomph
+
+- There are short example programs in `tests`.
+- Some things are implemented in Oomph. See `builtins.oomph` and `stdlib`.
+- The `self_hosted/` directory is an Oomph compiler written in Oomph.
+- Docs:
```

Comment:
```suggestion
```suggestion
Example code:
- There are short example programs in `tests`.
- Some things are implemented in Oomph. See `builtins.oomph` and `stdlib`.
- The `self_hosted/` directory is an Oomph compiler written in Oomph.

Docs:
``````